ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_losses.py

```diff
@@ -6,6 +6,74 @@
 from ivy_tests.test_ivy.helpers import handle_test
 
 
+# --- Helpers --- #
+# --------------- #
+
+
+@st.composite
+def _get_input_and_target(draw):
+    m = draw(st.integers(1, 5))
+    n = draw(st.integers(1, 5))
+    dtype_and_input = draw(
+        helpers.dtype_and_values(
+            num_arrays=1,
+            available_dtypes=["float32", "float64"],
+            min_value=-10,
+            max_value=10,
+            allow_inf=False,
+            min_num_dims=2,
+            max_num_dims=2,
```

Comment:
I believe this doesn't encompass the entire test space, as there's no need to confine the input shape to only two dimensions.